src/helpers/date-formatter.helper.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Returns the current date in the correct format for ingestion by the Salesforce sObject API.
```

Comment:
```suggestion
 * Returns the current date in the correct format for ingestion by the Salesforce Object API.
```

Reply:
Funnily enough this is actually not a typo - I think the s stands for salesforce?
https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_sobject.htm